atelier-openapi/Cargo.toml

```diff
@@ -18,7 +18,7 @@ all-features = true
 
 [dependencies]
 atelier_core = { version = "~0.2", path = "../atelier-core" }
-okapi = "0.4.0"
+okapi =  { git = "https://github.com/GREsau/okapi" }
```

Comment:
Can you add a comment as to why you aren't using the version number here?

Reply:
this is a temporary measure until a new release is published. See https://github.com/GREsau/okapi/issues/74. I'll chase that down before this PR gets out of draft

Reply:
Cool!

Reply:
got it thanks!